src/azure-cli/azure/cli/command_modules/backup/_params.py

```diff
@@ -293,6 +294,7 @@ def load_arguments(self, _):
 
     with self.argument_context('backup protection disable') as c:
         c.argument('delete_backup_data', arg_type=get_three_state_flag(), help='Option to delete existing backed up data in the Recovery services vault.')
+        c.argument('retain_recovery_points_as_per_policy', arg_type=get_three_state_flag(), options_list=['--retain-recovery-points-as-per-policy', '--retain-as-per-policy'], help='Switch parameter that specifies that existing recovery points should be retained for the duration specified by the backup policy')
```

Comment:
Can we define this parameter as `action='store_true'` instead of `arg_type=get_three_state_flag()`

Reply:
This should work, but it will introduce inconsistencies with previously defined flags that use the `arg_type=get_three_state_flag()` definition. IIUC, `action='store_true'` will break the `az command --flag <value>` workflow and only allow `az command --flag`.

Reply:
If that's alright, please resolve the conversation and merge.